src/styles/problem.scss

```diff
@@ -111,3 +111,34 @@ td.problem-cell {
 .problem-div-4 {
```

Comment:
I'd like something like this. Different class names with SCSS variables.

Reply:
@25lahari Remind. There are still some conversations need to be resolved.

Reply:
I sorry but not able to understand what u mean can you elaborate please.

Reply:
I mean I have defined some color variables in the beginning of this file. So you can reuse them in your css classes, and update the JSX like following,

```
<div className={'problem-div-' + x }
```

Reply:
I have updated the changes.